subprojects/packagefiles/cmark-gfm/meson.build

```diff
@@ -108,16 +100,43 @@ src = files(
     'src/xml.c',
 )
 
+cmark_gfm_extensions_src = files(
+    'extensions/autolink.c',
+    'extensions/core-extensions.c',
+    'extensions/ext_scanners.c',
+    'extensions/ext_scanners.h',
+    'extensions/strikethrough.c',
+    'extensions/table.c',
+    'extensions/tagfilter.c',
+    'extensions/tasklist.c',
+)
+
 # cmark-gfm has many unused parameter warnings, so we disable that warning
 # on compilers that support it.
 add_project_arguments(
     cc.get_supported_arguments('-Wno-unused-parameter'),
     language: 'c',
 )
 
-includes = include_directories('.', 'src', 'extensions')
-lib = library('cmark-gfm', src, include_directories: includes)
+cmark_gfm_includes = include_directories('.', 'src', 'extensions')
+cmark_gfm_lib = library(
+    'cmark-gfm',
+    cmark_gfm_src,
+    include_directories: cmark_gfm_includes,
+)
 cmark_gfm_dep = declare_dependency(
-    link_with: lib,
-    include_directories: includes,
+    link_with: cmark_gfm_lib,
+    include_directories: cmark_gfm_includes,
+)
+
+cmark_gfm_extensions_includes = include_directories('.', 'src', 'extensions')
+cmark_gfm_extensions_lib = library(
+    'cmark-gfm-extensions',
+    cmark_gfm_extensions_src,
+    include_directories: cmark_gfm_extensions_includes,
+    link_with: cmark_gfm_lib,
+)
+cmark_gfm_extensions_dep = declare_dependency(
+    link_with: cmark_gfm_extensions_lib,
+    include_directories: cmark_gfm_extensions_includes,
```

Comment:
I don't think we need a separate dependency for this. When installing with cmake globally, it installs a single pkg-config file "libcmark-gfm.pc" that links to `-lcmark-gfm -lcmark-gfm-extensions`.

Reply:
Fixed.